recipes/seqan/2.x.x/conanfile.py

```diff
@@ -0,0 +1,46 @@
+import os
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.28.0"
+
+
+class SeqanConan(ConanFile):
+    name = "seqan"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/seqan/seqan"
+    description = """
+SeqAn is an open source C++ library of efficient algorithms and data structures for the analysis of sequences 
+with the focus on biological data. Our library applies a unique generic design that guarantees high performance, 
+generality, extensibility, and integration with other libraries. 
+SeqAn is easy to use and simplifies the development of new software tools with a minimal loss of performance.
+"""
+    topics = ("conan", "cpp98", "cpp11", "cpp14", "cpp17",
+              "algorithms", "data structures", "biological sequences")
+    license = "BSD-3-Clause"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 14)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "seqan-library-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.download(
+            url="https://raw.githubusercontent.com/seqan/seqan/master/LICENSE",
+            filename=os.path.join(self._source_subfolder, "LICENSE")
+        )
```

Comment:
you can avoid this is you download the sources from github https://github.com/seqan/seqan/archive/seqan-v2.4.0.tar.gz

just a thought, not a requirement